// intern/lib/interfaces/bdd.js
(function (factory) {
    if (typeof module === "object" && typeof module.exports === "object") {
        var v = factory(require, exports);
        if (v !== undefined) module.exports = v;
    }
    else if (typeof define === "function" && define.amd) {
        define(["require", "exports", "./tdd"], factory);
    }
})(function (require, exports) {
    "use strict";
    Object.defineProperty(exports, "__esModule", { value: true });
    exports.getInterface = exports.afterEach = exports.beforeEach = exports.after = exports.before = exports.it = exports.describe = void 0;
    var tdd_1 = require("./tdd");
    Object.defineProperty(exports, "describe", { enumerable: true, get: function () { return tdd_1.suite; } });
    Object.defineProperty(exports, "it", { enumerable: true, get: function () { return tdd_1.test; } });
    Object.defineProperty(exports, "before", { enumerable: true, get: function () { return tdd_1.before; } });
    Object.defineProperty(exports, "after", { enumerable: true, get: function () { return tdd_1.after; } });
    Object.defineProperty(exports, "beforeEach", { enumerable: true, get: function () { return tdd_1.beforeEach; } });
    Object.defineProperty(exports, "afterEach", { enumerable: true, get: function () { return tdd_1.afterEach; } });
    function getInterface(executor) {
        var _a = tdd_1.getInterface(executor), suite = _a.suite, test = _a.test, before = _a.before, after = _a.after, beforeEach = _a.beforeEach, afterEach = _a.afterEach;
        return {
            describe: suite,
            it: test,
            before: before,
            after: after,
            beforeEach: beforeEach,
            afterEach: afterEach
        };
    }
    exports.getInterface = getInterface;
});
