// @theintern/digdug/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.Tunnel = exports.TestingBotTunnel = exports.SeleniumTunnel = exports.SauceLabsTunnel = exports.NullTunnel = exports.CrossBrowserTestingTunnel = exports.BrowserStackTunnel = void 0;
var BrowserStackTunnel_1 = require("./BrowserStackTunnel");
Object.defineProperty(exports, "BrowserStackTunnel", { enumerable: true, get: function () { return __importDefault(BrowserStackTunnel_1).default; } });
var CrossBrowserTestingTunnel_1 = require("./CrossBrowserTestingTunnel");
Object.defineProperty(exports, "CrossBrowserTestingTunnel", { enumerable: true, get: function () { return __importDefault(CrossBrowserTestingTunnel_1).default; } });
var NullTunnel_1 = require("./NullTunnel");
Object.defineProperty(exports, "NullTunnel", { enumerable: true, get: function () { return __importDefault(NullTunnel_1).default; } });
var SauceLabsTunnel_1 = require("./SauceLabsTunnel");
Object.defineProperty(exports, "SauceLabsTunnel", { enumerable: true, get: function () { return __importDefault(SauceLabsTunnel_1).default; } });
var SeleniumTunnel_1 = require("./SeleniumTunnel");
Object.defineProperty(exports, "SeleniumTunnel", { enumerable: true, get: function () { return __importDefault(SeleniumTunnel_1).default; } });
var TestingBotTunnel_1 = require("./TestingBotTunnel");
Object.defineProperty(exports, "TestingBotTunnel", { enumerable: true, get: function () { return __importDefault(TestingBotTunnel_1).default; } });
var Tunnel_1 = require("./Tunnel");
Object.defineProperty(exports, "Tunnel", { enumerable: true, get: function () { return __importDefault(Tunnel_1).default; } });
